traffic_portal/app/src/common/modules/form/deliveryService/form.deliveryService.DNS.tpl.html

```diff
@@ -251,7 +251,7 @@ <h3>Current Value</h3>
                         </label>
                         <div class="col-md-10 col-sm-10 col-xs-12">
                             <select id="serviceCategory" name="serviceCategory" class="form-control" ng-model="deliveryService.serviceCategory" ng-options="serviceCategory.id as serviceCategory.name for serviceCategory in serviceCategories">
-                                <option value="">Select...</option>
+                                <option [selected][hidden][disabled][value=""]>Select...</option>
```

Comment:
Oh, sorry, I gave it to you as a CSS selector. `option[selected][hidden][disabled][value=""]` is a selector that matches `<option selected hidden disabled value="">Anything here</option>` - the <kbd>[</kbd><kbd>]</kbd> just indicate that it's matching an attribute; it shouldn't actually appear as a part of the attribute names.